tensorflow/python/ops/image_ops.py

```diff
@@ -62,6 +62,7 @@
 @@non_max_suppression
 @@non_max_suppression_v2
 @@sample_distorted_bounding_box
+@@sample_distorted_bounding_box_v2
```

Comment:
Please don't expose the _v2 op directly. We need to register both kernels, but then only expose a single python op, which only generates the _v2 op. To do that, you currently have to mark both the v1 and v2 ops "hidden" in the BUILD file, and then write a python wrapper which calls the generated_image_ops._..._v2() directly (ignoring the linter that will complain about a protected symbol access).

We're working on improving this workflow but that's what it is for now.